ONSdigital/babbage
# Fix/cookie banner font size

### What

- Updated the font size to 18 for the cookies banner to align it with more recent copy patterns
- Fixed grammar issue on buttons
- Update `/cookies` to `/cookies/edit`

### How to review

- Check the markup for the new classes, ensure they're the most apt from Sixteens
- Check that new font size works with current design

### Who can review

Anyone bar me
